### DISCUSSION TITLE
Sponsorship

### DISCUSSION THREAD

**Discussion by thehesiod** [aio-libs/aiobotocore#878] [Ideas]
Hey guys, I figure we might as well let people help us in the work for maintaining this :) I'm going to add a sponsorship button with ppl with the most contributions these last couple years, I think that's @terrycain  and myself.  If people feel strongly about being added lets discuss here! :)

@aio-libs/aiobotocore-admins @terrycain

**Comment by thehesiod** [comment]
ok mine's live, wasn't too bad to activate

**Reply by terricain** [reply]
Nice, yeah doesn't look too difficult, will set it up tonight
